api/envoy/config/metrics/v2/stats.proto

```diff
@@ -201,7 +201,7 @@ message TagSpecifier {
     //
     //   {
     //     "tag_name": "envoy.cluster_name",
-    //     "regex": "^cluster\.((.+?)\.)"
+    //     "regex": "^cluster\\.((.+?)\\.)"
```

Comment:
Is this a docs only change? Will the docs be misleading vs. config users need to plugin?

Reply:
This should be a doc only change. The increased escaping enforcement in Pygment caught the issue with the existing code. What is there is not valid JSON. This is the only error produced in a full docs run with the new Python deps
![image](https://user-images.githubusercontent.com/246317/94955577-a20cef80-049f-11eb-94d3-09d3b9f8bc37.png)
![image](https://user-images.githubusercontent.com/246317/94955696-d385bb00-049f-11eb-9fb7-fe57c94ce9e1.png)